clientapi/routing/login.go

```diff
@@ -42,10 +43,15 @@ type flow struct {
 
 func passwordLogin() flows {
 	f := flows{}
-	s := flow{
-		Type: "m.login.password",
-	}
-	f.Flows = append(f.Flows, s)
+	f.Flows = append(f.Flows, flow{
+		Type: authtypes.LoginTypePassword,
+	})
+
+	// TODO: Add config option to disable
```

Comment:
Woops, left this in. 

I guess it's a good thing to discuss though - do want a config option for this? Otherwise we could just replace this all with an array literal